app-frontend/src/app/components/scenes/sceneDetailModal/sceneDetailModal.html

```diff
@@ -37,7 +37,7 @@ <h5 class="color-dark">Scene properties</h5>
             {{$ctrl.datasource ? $ctrl.datasource.name : 'Loading datasource'}}
           </dd>
           <dt ng-repeat-start="(field, value) in $ctrl.scene.filterFields">
-            {{field}}
+            {{field === 'acquisitionDate' ? field + ' (UTC)' : field}}
```

Comment:
Aren't time zones usually specified on the actual timestamp / date, not on the label?